extract_verbs.sh

```diff
@@ -3,10 +3,67 @@
 set -e
 
 cd ..
-# Download latest Italian Wikipedia dump
-wget http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2
+# Download latest Wikipedia dump in language of Chioce
+echo "Enter wikipedia dump language e.g. en OR english"
+read LANGCODE
+#convert lnaguge code to lower case
+LANGCODE="$(echo $LANGCODE | tr '[A-Z]' '[a-z]')"
+
+#looping to select language
```

Comment:
@marfox they reason why I didn't put a case statement is user can enter long name of the language or short code. For e.g. english or en. Making this is a switch-case statement makes code more lengthy (onle language two cases). This way it is much compact, do you still want to modify it as a switch-case ?